src/features/auth/pages/signup/SignupForm.native.test.tsx

```diff
@@ -227,71 +231,78 @@ describe('Signup Form', () => {
     expect(screen.getByText('Confirme ton adresse e-mail')).toBeOnTheScreen()
   })
 
-  it('should call logContinueSetEmail when clicking on next step from SetEmail', async () => {
-    renderSignupForm()
-
-    const emailInput = screen.getByPlaceholderText('tonadresse@email.com')
-    fireEvent.changeText(emailInput, 'email@gmail.com')
-    await act(() => fireEvent.press(screen.getByTestId('Continuer vers l’étape Mot de passe')))
-
-    expect(analytics.logContinueSetEmail).toHaveBeenCalledTimes(1)
-  })
+  describe('analytics', () => {
```

Comment:
Il y a les tests `should log analytics when clicking on close icon` et `should call logCancelSignup with Email when quitting after signup modal` qu'on peut mettre dans ce `describe`

Reply:
Bien vu !